common/lib/xblock/xblock_discussion/static/discussion_inline.js

```diff
@@ -0,0 +1,10 @@
+/* global DiscussionModuleView */
```

Comment:
I find this a bit confusing to declare this as a JS file under static when it is actually a template. Generally we put files under static that we are going to serve through the CDN. I would put this under a ``templates`` directory.